src/Components/Profile.js

```diff
@@ -25,8 +25,10 @@ function Profile({ profile, username }) {
             className="p-mr-2 p-mt-2"
           />
         </Avatar>
-        <h1 className="p-m-2">{name}</h1>
-        <h4 className="">({username})</h4>
+        <div className="p-m-2 p-d-flex">
+          <h1 className="p-m-0 p-r-3">{name}</h1>
```

Comment:
@SPARKY892 like this?

```suggestion
          <h1 className="p-m-2 p-r-3">{name}</h1>
```